addons/hr_expense/models/hr_expense.py

```diff
@@ -981,7 +859,7 @@ def _default_bank_journal_id(self):
         ('done', 'Done'),
         ('cancel', 'Refused')
     ], string='Status', index=True, readonly=True, tracking=True, copy=False, default='draft', required=True, help='Expense Report State')
-    payment_state = fields.Selection(selection=PAYMENT_STATE_SELECTION, string="Payment Status",
+    payment_state = fields.Selection(selection=AccountMove.payment_state.selection, string="Payment Status",
```

Comment:
I don't know if it makes sense to do so, but custom modules may add new `payment_state` values. What about getting the valid values dynamically?
```suggestion
    payment_state = fields.Selection(selection=lambda self: self.env["account.move.line"]._fields["payment_state"].selection, string="Payment Status",
```